Modules/SubModule_Import/xASL_imp_NII2BIDS_RunAnat.m

```diff
@@ -56,6 +56,12 @@ function xASL_imp_NII2BIDS_RunAnat(imPar, bidsPar, studyPar, subjectSessionLabel
             % Create the anatomical directory
             xASL_adm_CreateDir(anatOutDirLabel);
 
+            % Current scan name
+            [~, scanName] = xASL_fileparts([anatOutLabel,'_' iAnatType{1}]);
```

Comment:
The `,` here is confusing

Reply:
Why?

Reply:
on the right hand side - some delimiters are `' '`, some are `','`... Fixing...